### PR TITLE
EIP-1153 Review

### PR DESCRIPTION
**Pull request by Pandapip1**
Since it's not possible to review stuff in a PR that hasn't been changed, I've made this PR as a review of EIP-1153 to unblock #4942.

### CODE REVIEW

**Comment by Pandapip1** [comment]
@AlexeyAkhunov, @moodysalem would you mind reviewing?

File: EIPS/eip-1153.md [hunk 1]
```diff
@@ -47,13 +35,10 @@ Potential use cases enabled or improved by this EIP include:
...
 
-These opcodes are more efficient to execute than the `SSTORE` and `SLOAD` opcodes because the original value never needs to be loaded from storage (i.e. is always 0). 
-The gas accounting rules are also simpler, since no refunds are required.
+These opcodes are more efficient to execute than the `SSTORE` and `SLOAD` opcodes because the original value never needs to be loaded from storage (i.e. is always 0). The gas accounting rules are also simpler, since no refunds are required.
 
 ## Specification
-Two new opcodes are added to EVM, `TLOAD` (`0xb3`) and `TSTORE` (`0xb4`). Note that previous drafts of this EIP specified 
-the values `0x5c` and `0x5d` respectively for `TLOAD` and `TSTORE` respectively, but these have been modified so as not
-to conflict with other draft EIPs.
+Two new opcodes are added to EVM, `TLOAD` (`0xb3`) and `TSTORE` (`0xb4`). Note that previous drafts of this EIP specified the values `0x5c` and `0x5d` respectively for `TLOAD` and `TSTORE` respectively, but these have been modified so as not to conflict with other draft EIPs.
```
Comment (moodysalem) [inline EIPS/eip-1153.md:41]: double respectively oops

File: EIPS/eip-1153.md [hunk 2]
```diff
@@ -63,35 +48,20 @@ They use the same arguments on stack as `SLOAD` (`0x54`) and `SSTORE` (`0x55`).
 
 Addressing is the same as `SLOAD` and `SSTORE`. i.e. each 32-byte address points to a unique 32-byte word.
 
-Gas cost for `TSTORE` is the same as a hot dirty `SSTORE` (original value is not new value and is not current value, currently 100 gas), 
-and gas cost of `TLOAD` is the same as a hot `SLOAD` (value has been read before, currently 100 gas). Gas cost cannot be on par with memory access due to transient storage's interactions with reverts.
+Gas cost for `TSTORE` is the same as a hot dirty `SSTORE` (original value is not new value and is not current value, currently 100 gas), and gas cost of `TLOAD` is the same as a hot `SLOAD` (value has been read before, currently 100 gas). Gas cost cannot be on par with memory access due to transient storage's interactions with reverts.
```
Comment (moodysalem) [inline EIPS/eip-1153.md:51]: maybe "hot dirty" > "warm"

File: EIPS/eip-1153.md [hunk 3]
```diff
@@ -127,22 +92,15 @@ A map plus an auxiliary writeahead list is one implementation:
 - On revert a search is made for the matching call marker on the list. All entries on the list are reverted and the list cleared to that marker.
 
 ## Security Considerations
-`TSTORE` presents a new way to allocate memory on a node with linear cost. In other words, each TSTORE allows the developer 
-to store 32 bytes for 100 gas, excluding any other required operations to prepare the stack. Given 30 million gas, the
-maximum amount of memory that can be allocated using TSTORE is:
+`TSTORE` presents a new way to allocate memory on a node with linear cost. In other words, each TSTORE allows the developer to store 32 bytes for 100 gas, excluding any other required operations to prepare the stack. Given 30 million gas, themaximum amount of memory that can be allocated using TSTORE is:
```
Comment (moodysalem) [inline EIPS/eip-1153.md:95]: space missing themaximum

**Review by moodysalem — APPROVED** [review]
i will add fixes

**Pull request merged**